themes/janeswalk/js/components/pages/WalkPage/WalkTeam.jsx

```diff
@@ -20,13 +18,12 @@ const WalkTeam = ({team}) => {
   let teamMembers = team.map((member, i)=> (
     <article key={i}>
       <header>
-        <h3>{member['name-first']} {member['name-last']}</h3>
-        <h4>{member['role']}</h4>
+        <h3>{member['name-first']} {member['name-last']}, <span className="walkTeamMemberRole">{member['role']}</span></h3>
```

Comment:
Some members don't have both names, but they'll get an extra space in this component. Shouldtrim these off. I also don't like having more `{}` than are necessary, since React renders and compares them separately before building into its virtual DOM. I'm guessing it only looks like this here because it's following the same pattern as the PHP template. `(member['name-first'] + ' ' + member['name-last']).trim()`